### DISCUSSION TITLE
Sequel Ace Adding CR on CVS import

### DISCUSSION THREAD

**Discussion by fpakay** [Sequel-Ace/Sequel-Ace#1599] [General]
When I import a.csv file, carriage returns are added to a text field but not any other type. Why? I am using the same options I used in sequel pro.

**Comment by samuelbradshaw** [comment]
I don't know if your situation is the same as mine, but this happens to me when I export a CSV file from Apple Numbers. If a text-type field is the last column in the CSV, that last column imports with extra new lines. There are two workarounds I've found:
1. Reorder the columns so that an int or an ignored field is the last column in the CSV.
2. Change the "Lines terminated by:" setting to "\r\n" before importing (see screenshot). Unfortunately, it always seems to reset, so I have to remember to change it every time.

<img width="1001" alt="Screen Shot 2022-11-05 at 8 32 21 PM" src="https://user-images.githubusercontent.com/7323052/200151237-514a5ea0-40f6-411b-b1dd-75f41c1275f8.png">

**Reply by fpakay** [reply]
Thanks for the suggestion. I already tried "\r\n". The column with the return being added is the last column with data. There is a blank field following it. I will reorder the columns and try. Thanks again.
